packages/reaction-product-variant/client/templates/products/productDetail/productDetail.js

```diff
@@ -199,17 +199,20 @@ Template.productDetail.events({
       template.$(".title-edit-input").focus();
     }
     const variants = ReactionProduct.getVariants(self._id);
-    for (let variant of variants) {
-      let index = _.indexOf(variants, variant);
+    variants.forEach((variant, index) => {
```

Comment:
Because, `For - of` - is for Objects, not for Arrays. For arrays es6 provides to as: `forEach`, `map`, `reduce`, `filter`, etc. Look at the [list of available Array methods](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach).
So in my opinion it is native to use Array iteration methods to iterate through it.